flixel/text/FlxBitmapText.hx

```diff
@@ -1461,7 +1462,7 @@ class FlxBitmapText extends FlxSprite
 	 * Set border's style (shadow, outline, etc), color, and size all in one go!
 	 * 
 	 * @param	Style outline style
-	 * @param	Color outline color in flash 0xRRGGBB format
+	 * @param	Color outline color in flash 0xAARRGGBB format
```

Comment:
Hm, I guess is a conscious limitation to not have alpha for FlxText's borderColor because of the way it's drawn (copying the original BitmapData to slightly offsetted positions). Would look weird with the overlaps.

Might still be possible to do if the outline is first copied entirely with alpha = 1, and then the transparency is applied in a separate draw call. Would be more expensive though?